src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -173,6 +173,12 @@ public virtual string CustomMethod
         #region Proxy
 
         /// <summary>
+        /// gets or sets the NoProxy property
+        /// </summary>
+        [Parameter]
+        public virtual SwitchParameter NoProxy { get; set; }
```

Comment:
I'm confused about the switch, all the more so in a separate parameter set. I would prefer not to complicate and just make `-Proxy $null`